dot/network/service.go

```diff
@@ -333,12 +333,12 @@ main:
 			o := s.host.bwc.GetBandwidthTotals()
 			err := telemetry.GetInstance().SendMessage(telemetry.NewBandwidthTM(o.RateIn, o.RateOut, s.host.peerCount()))
 			if err != nil {
-				logger.Debug("problem sending system.interval telemetry message", "error", err)
+				logger.Warn("problem sending system.interval telemetry message", "error", err)
```

Comment:
can we leave these as debug? I saw these getting spammed before sometimes lol. same for the others

Reply:
addressed